src/overlays/actors/ovl_Door_Warp1/z_door_warp1.c

```diff
@@ -502,10 +510,11 @@ void func_808B98A8(DoorWarp1* this, PlayState* play) {
 
 void func_808B9B30(DoorWarp1* this, PlayState* play) {
     if (fabsf(this->dyna.actor.xzDistToPlayer) >= 60.0f) {
-        if (func_808B849C(this, play)) {
-            this->unk_1A0 = (DmHina*)Actor_SpawnAsChild(
-                &play->actorCtx, &this->dyna.actor, play, ACTOR_DM_HINA, this->dyna.actor.world.pos.x,
-                this->dyna.actor.world.pos.y, this->dyna.actor.world.pos.z, 0, 0, 0, func_808B849C(this, play) - 1);
+        if (DoorWarp1_GetRemains(this, play)) {
```

Comment:
The function doesn't return a bool, so an explicit check against zero is preferred.
(I know it wasn't you who made the check this way, but it is better to clean it up now that we noticed)
```suggestion
        if (DoorWarp1_GetRemains(this, play) != 0) {
```

Reply:
I'll do this because of your preference, but IMO it's not less clear without that.